src/sql/inspect.py

```diff
@@ -73,6 +75,170 @@ def __init__(self, name, schema, conn=None) -> None:
         self._table_txt = self._table.get_string()
 
 
+@modify_exceptions
+class TableDescription(DatabaseInspection):
+    """
+    Generates descriptive statistics.
+
+    Descriptive statistics are:
+
+    Count - Number of all non empty values
+
+    Mean - Mean of the values
+
+    Max - Maximum of the values in the object.
+
+    Min - Minimum of the values in the object.
+
+    STD - Standard deviation of the observations
+
+    25h, 50h and 75h percentiles
+
+    Unique - Number of unique values
+
+    Top - The most frequent value
+
+    Freq - Frequency of the top value
+
+    """
+
+    def __init__(self, table_name, schema=None, config=None, user_ns=None) -> None:
+
+        if schema:
+            table_name = f"{schema}.{table_name}"
+
+        result_table_columns = sql.run.run(
```

Comment:
using run might break since it's output is affected by the user's settings, let's create a `run_raw` function that calls `conn.session.execute` and returns the results without any extra logic (see the implementation of `run`)